src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/notification/DefendantResponseCaseHandedOfflineApplicantNotificationHandler.java

```diff
@@ -65,7 +64,7 @@ private CallbackResponse notifyDefendantSolicitorForCaseHandedOffline(CallbackPa
     public Map<String, String> addProperties(CaseData caseData) {
         return Map.of(
             CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
-            SOLICITOR_REFERENCE, "defendant solicitor"
+            SOLICITOR_REFERENCE, "claimant solicitor"
```

Comment:
is this the reference we capture duering claim creation journey?

Reply:
same as above